game/scripts/vscripts/items/meteor_hammer.lua

```diff
@@ -0,0 +1,203 @@
+LinkLuaModifier("modifier_generic_bonus", "modifiers/modifier_generic_bonus.lua", LUA_MODIFIER_MOTION_NONE)
+LinkLuaModifier("modifier_item_meteor_hammer_thinker", "items/meteor_hammer.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier("modifier_item_meteor_hammer_damage_over_time", "items/meteor_hammer.lua", LUA_MODIFIER_MOTION_NONE)
+
+item_meteor_hammer = class(ItemBaseClass)
+item_meteor_hammer_1 = item_meteor_hammer
+item_meteor_hammer_2 = item_meteor_hammer_1
+item_meteor_hammer_3 = item_meteor_hammer_1
+item_meteor_hammer_4 = item_meteor_hammer_1
+item_meteor_hammer_5 = item_meteor_hammer_1
+
+function item_meteor_hammer:OnSpellStart()
+
+  local caster = self:GetCaster()
+
+  caster:EmitSound("DOTA_Item.MeteorHammer.Channel")
+
+  caster:StartGesture(ACT_DOTA_TELEPORT)
+
+  if IsServer() then
+    --dehardcode ---------------------------------------
+    self:CreateVisibilityNode(self:GetCursorPosition(),self:GetSpecialValueFor("impact_radius"), 3.8 )
+
+    --Particle that surrounds caster
+    self.channel_particle_caster = ParticleManager:CreateParticle("particles/items4_fx/meteor_hammer_cast.vpcf", PATTACH_ABSORIGIN, self:GetCaster())
+    --Particle that surrounds meteor_hammer's aoe.
+    self.channel_particle = ParticleManager:CreateParticleForTeam("particles/items4_fx/meteor_hammer_aoe.vpcf", PATTACH_CUSTOMORIGIN, self:GetCaster(), self:GetCaster():GetTeam())
+    ParticleManager:SetParticleControl(self.channel_particle, 0, self:GetCursorPosition())
+    ParticleManager:SetParticleControl(self.channel_particle, 1, Vector(self:GetSpecialValueFor("impact_radius"), 0, 0))
+
+
+
+  end
+end
+
+function item_meteor_hammer:GetChannelAnimation()
```

Comment:
I believe you said this didn't work. If that's the case, you might as well remove the function.